### ISSUE TITLE
Fibaro FGMS001 Zwave Plus Motion Sensor with Device Type 801 not working properly

### ISSUE THREAD

**Issue by anordvall** [openhab/openhab#4274]
[openhab debug.log.txt](https://github.com/openhab/openhab/files/211904/openhab.debug.log.txt)

The new ZWave Plus (Device Type 801) of Fibaro FGMS001 does not seem to get initialized properly. I do not have any problems with the earlier generation (Device Type 800).

Controller: Aeon Labs Z-Stick
OS: Raspberry Pi 3 with Raspian OS.
Z-Wave addon: Downloaded from cloudbees today April 10th.

Please find the debug log attached. The Fibaro Motion Sensor is NODE 7. I reset the node and included it in the controller before starting openHAB.

Really appreciate if you can have a look at this one @cdjackson . My brother have the exact same problem but with a Z-Stick connected to a PC running Windows, so at least I'm not the only one seeing it.

I can also mention that the device does not work properly either. When motion is detected, openHAB logs it as receiving a command class ALARM (same as what is logged when you shake the device to trigger Tamper Alarm)

**Comment by cdjackson** [comment]
We need to create a new version in the database - this is different. The immediate problem is that parameter 26 is no longer included in version 3.2, but there may be other issues.

I’ve created a new entry in the database -:
http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/328 <http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/328>

Please can you upload your XML file to this (hopefully it will detect the correct version) and then ensure that all the parameters and association groups are correct as per the manual (I’ve loaded the manual from the Fibaro site for version 3.2).

Any problems, please let me know.

Chris

**Comment by anordvall** [comment]
There is no XML generated yet. Do you mean the XLM generated once I try with the new DB?

**Comment by cdjackson** [comment]
Hmmm - good point. Ok - if you can get the configuration corrected we can update the database in the binding, then it should create the XML...

**Comment by anordvall** [comment]
You lost me. Might be because I had a couple of glasses of red wine :) Can you point me in the right direction of what I should do? Many thanks!

**Comment by cdjackson** [comment]
Are you registered on the database site? If not, first register and then I’ll give you access. The you should make sure that all the parameters are correct - I know that parameter 26 needs to be deleted, but maybe there are other parameters that are different in this version. There’s probably not many changes, but it would be good to get it correct.

Once you’re happy this is ok, then email me and I’ll update the database - it should then work, and then it should create the XML.

Enjoy the wine :)

**Comment by anordvall** [comment]
I assume "the database site" is the cd-jacksson site you linked? I've now registered an account (btw, it mailed me my password in plain text?!) :)

**Comment by cdjackson** [comment]
Yes, and I’ve just given you edit access…

I thought I’d disabled the sending of passwords - it’s a very annoying default feature of Joomla!  I’ll try and find the setting again - thanks.

**Comment by anordvall** [comment]
I'll go through the device manual parameter by parameter tomorrow. Thanks as always for all your help Chris!

**Comment by anordvall** [comment]
@cdjackson - Quite some changes compared to the previous version. I went through the manual parameter by parameter and attached are my findings for the ones that have changed.

[Fibaro motion sensor diff.txt](https://github.com/openhab/openhab/files/222656/Fibaro.motion.sensor.diff.txt)

The manuals I compared was:
OLD: http://manuals.fibaro.com/content/manuals/en/FGMS-001/FGMS-001-EN-A-v1.01.pdf
NEW: http://products.z-wavealliance.org/MarketCertification/File?folder=&filename=MarketCertificationFiles/1146/FGMS-001-EN-A-v1.02.pdf

**Comment by cdjackson** [comment]
Wow - that's a comprehensive check 👍 

Can you update the new device in the database with the changes?

http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/328 

Thanks
Chris

**Comment by anordvall** [comment]
I will be away for 2 weeks now but after that I will give it a shot unless someone else beats it to me, which I hope someone do :)

**Comment by xdxoxkx** [comment]
I am new and testing Openhab with this sensor ... I got Version 4 ... the behavior is quite strange ... hope for some updates ... thx

**Comment by tiger-seo** [comment]
i have the same type of the device and also can't initialize it with openhab

**Comment by cdjackson** [comment]
What version are you using? What device type/id does the device report?

**Comment by xdxoxkx** [comment]
From openHAB Admistration Interface:

Version: 4
Product: FGMS001 Motion Sensor
Device Type: 801
Device ID: 1001
Application Version: 3.2
Protocol Version: 4.5

... and ...
Generic Device Class: Garage_Door
Specific Device Class: Simple_Garage_Door

**Comment by cdjackson** [comment]
And what version of the binding are you using?

**Comment by xdxoxkx** [comment]
At the moment ...
-> org.openhab.binding.zwave-1.9.0-fibaro-eye-hack.jar

before:
org.openhab.binding.zwave-1.9.0-SNAPSHOT.jar and before 
org.openhab.binding.zwave-1.8.1.jar

**Comment by cdjackson** [comment]
What does "the behavior is quite strange" mean?

If the problem is still related to the database issues that were explained above, then it would be appreciated if someone can update the database with the changes for the new device -:

http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/328

**Comment by tiger-seo** [comment]
here is my zwave log https://gist.github.com/tiger-seo/48d4745709095fb91e9c220b6cb016ac

**Comment by cdjackson** [comment]
Thanks - so it looks like the same problem I already identified……..

**Comment by anordvall** [comment]
@cdjackson I just found some time to dig into this, but I believe I still don't have edit rights? When I login and browse to:

http://www.cd-jackson.com/index.php/zwave/zwave-device-database/zwave-device-list/devicesummary/328

It says:

"Please note that you must contact the administrator to get edit rights to the database once you have registered."

... and I don't find any edit button anywhere, so I assume I don't have access yet :)

**Comment by cdjackson** [comment]
It looks like you should have access - do you see the edit buttons on the device summary page?

If not, please email me your login name and I’ll triple check...

**Comment by anordvall** [comment]
You mean the Device Overview page, shown after clicking on a device in the devicelist? Nope, no Edit buttons... my username is "anordvall". I've tried in both IE and Chrome.

**Comment by cdjackson** [comment]
Ok - should be good now (there’s a few people called Andreas :) - I got the wrong ones - sorry ).

**Comment by anordvall** [comment]
Looks better now :) Ok I will give this a try...

**Comment by anordvall** [comment]
Chris, I updated a lot of parameters and it said they all waited for review. Then I added parameter 25 and now it only says that parameter 25 is waiting for review. Can you check so not all my other changes got lost?

**Comment by anordvall** [comment]
Never mind... seems I accidentally added param 25 to the old device database entry. Trying to correct this ... :)

**Comment by cdjackson** [comment]
You probably won’t be able to change this - I can delete it though if you set its label to DELETE ME or something like that.

**Comment by anordvall** [comment]
It's not in the list so I can't edit it (?). Anyway, I pressed request for review... so please delete it right away so it's not forgotten about.

**Comment by anordvall** [comment]
Hmm... I now see parameter 25 ended up correctly in device 328, but the change message ended up incorrectly in 115 :) Seems there is a bug in the system..

**Comment by cdjackson** [comment]
Seems so :(  Did you swap between browser windows or something? Sometimes this sort of thing happens if you have two sessions open?

Looks ok though - thanks.  I’ll get this added tonight, so let me know how it goes…

Just to confirm - you need this on OH1 right?

**Comment by anordvall** [comment]
Yes I have both the old and new device config open at the same time, and I probably clicked Edit on the old one just to check something, and then went back.

Yes I need it for OH1. I still have some parameters to update so give me another hour :)

**Comment by cdjackson** [comment]
Ok - no probs - just let me know when you’re done…

**Comment by anordvall** [comment]
All done. Note that parameter 28 and 29 looks almost identical but it's not a mistake. They only differ slightly when mentioning 3rd and 5th association groups. Feel free to edit the labels if you think they should differ. Also notice that two options in parameter 24 should be deleted (marked as DELETE ME).

**Comment by cdjackson** [comment]
Thanks - I’ve deleted the two options…

**Comment by anordvall** [comment]
Chris, I downloaded the new zwave build from cloudbees today and tried it out. The Fibaro Motion sensor then was identified (still as a garage door though?), an XML was created and I saw the updated configuration parameters from the new DB entry. Still, the sensor didn't work as expected... when motion was detected it sent an alarm (whyyy?) as it did previously, so no improvement. Anyway, I excluded the device, reset it and included it again. Now the device was added using the old database entry! Perhaps 0801:1001 must be removed from the older product version (DB entry 115)? I've attached debug log (NODE 13) and node13.xml from this last attempt. Appreciate if you can have a look at them.

[openhab.log.txt](https://github.com/openhab/openhab/files/269076/openhab.log.txt)
[node13.xml.txt](https://github.com/openhab/openhab/files/269077/node13.xml.txt)

**Comment by cdjackson** [comment]
I don’t see anything obviously wrong.

Changing the database won’t change what the device sends, so it won’t change the fact that it sends an alarm.

Don’t worry about the ‘garage door’ thing - that’s just a name and isn’t relevant.

Why do you think it was detected as the wrong database entry?

**Comment by anordvall** [comment]
Ah, it must have been my browser that cached the old page because I could see suddenly see all old parameters after my last include. Now without having done any change, I see the new parameters again.

Anyway, any idea why it sends an alarm at motion detection? I can't figure out how to solve this... Anyone else reading this thread have any idea? I'm almost thinking about returning the devices and buy old non-plus ones.

**Comment by cdjackson** [comment]
Some devices send alarms (notifications) - that’s quite normal.

**Comment by anordvall** [comment]
Hi Chris. I realized there are also changes to the association groups compared to previous version. This explains why I currently get proper motion events (though don't understand why I get double events) when setting the Tamper association group (which is nowadays Motion association group). Please review especially the "max nodes" as I'm not sure it should be set to 5 especially in the 5th association group.

Also, note that I have not even reviewed the End Points configuration as I have no idea what that should look like.

**Comment by cdjackson** [comment]
Thanks. I think it looks ok. I removed the ‘controller’ tag from group 3 as this will cause multiple notifications (at least it does on other Fibaro devices if you use the lifeline group, and have other groups linked to the controller.

The endpoints should be ok - this gets read from your XML so should be directly from the device. It’s possible though that it might miss some stuff as I’ve recently added a feature to ensure we get all command classes logged, which wasn’t the case previously… For OH1 though, it’s probably fine.

I’ll try and remember to update the OH1 database tonight.

**Comment by anordvall** [comment]
Ah, I noticed #3 had Controller since before. I meant to remove it but I forgot :)
I changed from "garage" in Endpoints now as well. Please review =)

**Comment by anordvall** [comment]
Please do... I'm eager to test this now :)

**Comment by cdjackson** [comment]
Hmm - not a good idea to change the class - it should stay as Garage for now. Ultimately I will change the name but it needs to be changed in many places at the same time as this name, even though it is wrong, is used to link things together (the actual name doesn’t matter, but it needs to be consistent in the database and the binding and the XML files).

**Comment by anordvall** [comment]
Ok I see. Please decline my change.

**Comment by cdjackson** [comment]
Have done - I need to update all this as I do understand it’s confusing to people…

**Comment by anordvall** [comment]
Chris, even though the association group names are wrong for me with my current binding version, I now set Group 1 to 0, Group 2 to 1 and Group 3 to 0. When I trigger motion this is what is logged:

```
13:17:21.450 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 09 00 04 08 0D 03 20 01 FF 2A
13:17:21.451 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
13:17:21.451 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
13:17:21.452 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 09 00 04 08 0D 03 20 01 FF 2A
13:17:21.453 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 09 00 04 08 0D 03 20 01 FF 2A
13:17:21.453 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = ApplicationCommandHandler (0x04), type = Request (0x00), payload = 08 0D 03 20 01 FF
13:17:21.454 [DEBUG] [ApplicationCommandMessageClass:40   ] - NODE 13: Application Command Request (ALIVE:PING)
13:17:21.455 [DEBUG] [ApplicationCommandMessageClass:58   ] - NODE 13: Incoming command class BASIC
13:17:21.455 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:73   ] - NODE 13: Received Basic Request
13:17:21.456 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:77   ] - NODE 13: Basic Set sent to the controller will be processed as Basic Report
13:17:21.456 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:106  ] - NODE 13: Basic report, value = 0xFF
13:17:21.457 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveCommandClassValueEvent
13:17:21.457 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
13:17:21.458 [DEBUG] [.z.internal.ZWaveActiveBinding:466  ] - NODE 13: Got a value event from Z-Wave network, endpoint = 0, command class = BASIC, value = 255
13:17:21.459 [DEBUG] [ApplicationCommandMessageClass:89   ] - Transaction not completed: node address inconsistent.
13:17:21.487 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 09 00 04 00 0D 03 20 01 FF 22
13:17:21.488 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
13:17:21.488 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
13:17:21.489 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 09 00 04 00 0D 03 20 01 FF 22
13:17:21.490 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 09 00 04 00 0D 03 20 01 FF 22
13:17:21.490 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = ApplicationCommandHandler (0x04), type = Request (0x00), payload = 00 0D 03 20 01 FF
13:17:21.491 [DEBUG] [ApplicationCommandMessageClass:40   ] - NODE 13: Application Command Request (ALIVE:PING)
13:17:21.491 [DEBUG] [ApplicationCommandMessageClass:58   ] - NODE 13: Incoming command class BASIC
13:17:21.491 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:73   ] - NODE 13: Received Basic Request
13:17:21.492 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:77   ] - NODE 13: Basic Set sent to the controller will be processed as Basic Report
13:17:21.492 [DEBUG] [z.i.p.c.ZWaveBasicCommandClass:106  ] - NODE 13: Basic report, value = 0xFF
13:17:21.492 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveCommandClassValueEvent
13:17:21.493 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
13:17:21.493 [DEBUG] [.z.internal.ZWaveActiveBinding:466  ] - NODE 13: Got a value event from Z-Wave network, endpoint = 0, command class = BASIC, value = 255
13:17:21.494 [DEBUG] [ApplicationCommandMessageClass:89   ] - Transaction not completed: node address inconsistent.
```

Any idea why the motion sensor (NODE 13) sends a duplicate event?

Also, when "Controller" is set in an association group config (as you earlier noticed in group 3)... does it only mean the controller is by default assigned to the association group, or does it mean that it is forced to the group and can not be removed? Just for my understanding.

**Comment by cdjackson** [comment]
I’m not sure why it would send two notifications. Maybe it’s a config setting, or maybe not - I’m not sure…

Regarding the controller setting in the database - this just means that it will be set there by default - you can remove it, although when the binding restarts, it might restore this, so to some degree it will be forced...

**Comment by RosaEinhorn** [comment]
Hi everybody,

I am having problems using the same device (version: 4) with habmin.
Yesterday, the device worked for some hours and sent some data to the controller, than it changed to grey in habmin and finally to red. However, it is still sending data to the controller (e.g. when there is motion). But there are problems with the data:

`NODE 3: Got a value event from Z-Wave network, endpoint = 0, command class = SENSOR_BINARY, value = 255
Transaction not completed: node address inconsistent.`

I am also getting this error:
`2016-06-03 12:54:27.382 [ERROR] [.s.container.ContainerResponse] - The RuntimeException could not be mapped to a response, re-throwing to the HTTP container
java.lang.NullPointerException: null
	at org.openhab.binding.zwave.internal.config.ZWaveConfiguration.getConfiguration(ZWaveConfiguration.java:127) ~[bundlefile:na]
	at org.openhab.io.habmin.services.zwave.ZWaveConfigResource.getConfig(ZWaveConfigResource.java:73) ~[na:na]
	at sun.reflect.GeneratedMethodAccessor47.invoke(Unknown Source) ~[na:na]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_60]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_60]
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$ResponseOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:205) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.model.method.dispatch.ResourceJavaMethodDispatcher.dispatch(ResourceJavaMethodDispatcher.java:75) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.uri.rules.HttpMethodRule.accept(HttpMethodRule.java:302) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.uri.rules.ResourceClassRule.accept(ResourceClassRule.java:108) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.uri.rules.RightHandPathRule.accept(RightHandPathRule.java:147) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.uri.rules.RootResourceClassesRule.accept(RootResourceClassesRule.java:84) ~[jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1542) [jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.application.WebApplicationImpl._handleRequest(WebApplicationImpl.java:1473) [jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1419) [jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.server.impl.application.WebApplicationImpl.handleRequest(WebApplicationImpl.java:1409) [jersey-server-1.18.1.jar:1.18.1]
	at com.sun.jersey.spi.container.servlet.WebComponent.service(WebComponent.java:409) [jersey-servlet-1.18.1.jar:1.18.1]
	at com.sun.jersey.spi.container.servlet.ServletContainer.service(ServletContainer.java:540) [jersey-servlet-1.18.1.jar:1.18.1]
	at com.sun.jersey.spi.container.servlet.ServletContainer.service(ServletContainer.java:715) [jersey-servlet-1.18.1.jar:1.18.1]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848) [javax.servlet_3.0.0.v201112011016.jar:na]
	at org.atmosphere.util.AtmosphereFilterChain.doFilter(AtmosphereFilterChain.java:135) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.util.AtmosphereFilterChain.invokeFilterChain(AtmosphereFilterChain.java:96) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.handler.ReflectorServletProcessor$FilterChainServletWrapper.service(ReflectorServletProcessor.java:317) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.handler.ReflectorServletProcessor.onRequest(ReflectorServletProcessor.java:160) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.cpr.AsynchronousProcessor.action(AsynchronousProcessor.java:206) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.cpr.AsynchronousProcessor.suspended(AsynchronousProcessor.java:105) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.container.Jetty7CometSupport.service(Jetty7CometSupport.java:57) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.container.JettyAsyncSupportWithWebSocket.service(JettyAsyncSupportWithWebSocket.java:67) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.cpr.AtmosphereFramework.doCometSupport(AtmosphereFramework.java:2076) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.cpr.AtmosphereServlet.doPost(AtmosphereServlet.java:211) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at org.atmosphere.cpr.AtmosphereServlet.doGet(AtmosphereServlet.java:197) [atmosphere-runtime-2.2.5.jar:2.2.5]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735) [javax.servlet_3.0.0.v201112011016.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848) [javax.servlet_3.0.0.v201112011016.jar:na]
	at org.eclipse.equinox.http.servlet.internal.ServletRegistration.service(ServletRegistration.java:61) [org.eclipse.equinox.http.servlet_1.1.300.v20120522-1841.jar:na]
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.processAlias(ProxyServlet.java:128) [org.eclipse.equinox.http.servlet_1.1.300.v20120522-1841.jar:na]
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.service(ProxyServlet.java:68) [org.eclipse.equinox.http.servlet_1.1.300.v20120522-1841.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848) [javax.servlet_3.0.0.v201112011016.jar:na]
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:598) [org.eclipse.jetty.servlet_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:486) [org.eclipse.jetty.servlet_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:231) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1065) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:413) [org.eclipse.jetty.servlet_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:192) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:999) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:117) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:250) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:149) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:111) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.Server.handle(Server.java:350) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:454) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:890) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:944) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:630) [org.eclipse.jetty.http_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:230) [org.eclipse.jetty.http_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:77) [org.eclipse.jetty.server_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:606) [org.eclipse.jetty.io_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:46) [org.eclipse.jetty.io_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:603) [org.eclipse.jetty.util_8.1.3.v20120522.jar:8.1.3.v20120522]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:538) [org.eclipse.jetty.util_8.1.3.v20120522.jar:8.1.3.v20120522]
	at java.lang.Thread.run(Thread.java:745) [na:1.7.0_60]`

Would be cool if someone could help. :-)

**Comment by cdjackson** [comment]
What version of ZWave are you using?

**Comment by RosaEinhorn** [comment]
ZWave Binding is 1.8.3

**Comment by cdjackson** [comment]
Please use the latest snapshot of the zwave binding and see if this still occurs - I think it will fix the problem you had initially at least.

**Comment by RosaEinhorn** [comment]
Hi,
thanks a lot for your quick reply! :-)
I'm now using 1.9 SNAPSHOT and after some time of waiting (like 30 minutes) the Node switched from grey to green again and is sending some data - let's see if it lasts longer.

Node 1 (the controller) needs to be set in the group "Device Status" to receive data, correct?
It is already set in the group "Controller Updates".

Thanks a lot again.

**Comment by tiger-seo** [comment]
hello. i've also been able to get FGMS (device type 801) green in habmin and it's working, at least for few hours after openhab started, but randomly some issue arise with message like "Protocol error (OOF). Got 0x03. Sending NAK." and after that openhab does not receive from FGMS, unless i get openhab restarted. please, see my log https://gist.github.com/tiger-seo/2dc5abf88c26f1fa3ade6774be05e2b1 at 2016-07-26 01:29:38
thank you much

**Comment by tiger-seo** [comment]
just updated binding and i think it got even worse :(
 https://gist.github.com/tiger-seo/83575a90b75614b40b00f341782426dd

**Comment by cdjackson** [comment]
I don't think your problem is related to this issue. Your problem appears to be that your COM port is missing characters somehow and this means that packets are not being processed correctly. You need to resolve this first.

**Comment by tiger-seo** [comment]
@cdjackson so i restarted OH, it works (sends movement events) for 4-5 times, but then out-of-the-blue protocol error,  and you think this is problem on my hardware ?
pls look at new log https://gist.github.com/tiger-seo/40783597b4ac4393d8fd805c85e51275

**Comment by cdjackson** [comment]
The root cause of the OOF errors is that the data being received appears to be corrupted. The binding is reading a much larger frame than is realistic, and we then get the OOF errors as it is then trying to frame sync again. The actual issue (I think) is that the system is dropping a byte in the frame, this causes the binding to read the first byte in the next frame. The first frame (with the dropped by) is corrupt, and the next frame(s!) are also in error as we are now in the OOF condition.

Unfortunately the serial protocol is not good at resyncing since SOF is not escaped. Maybe the binding could add further checks to make it more tolerant to such errors, but you should try and solve the root as anything else that the binding adds will not solve this. 

What hardware are you running? Is this running on a VM or an IP serial connection or anything?

**Comment by cdjackson** [comment]
I don’t think it’s really a problem with the zwave binding - I certainly wouldn’t say it was unstable. It’s likely to be an issue that the Pi can’t keep up with the hardware, and it drops data on the serial port. The ZWave API isn’t especially tolerant to such errors. I could add a timer to help the resync process, but as I said earlier, you really need to deal with the root cause of dropping data on the USB since anything else won’t stop the fundamental errors.

I would suggest that if you’re totally maxing out the Pi that you should consider if you can rewrite your scripts to make them more efficient.

**Comment by pallemannen** [comment]
Hi Chris

I have one of those FGMS-001 - and I can't get it operational using org.openhab.binding.zwave-1.9.0-SNAPSHOT (build 201608110110 according to Habmin).

Habmin says Manufacturer ID, Device IDand Device Type "UNKNOWN". 
Version is 4, Listening is false, Routing is true, Power is UNKNOWN.
Base Device Class is ROUTING_SLAVE, Generic Device Class is GARAGE_DOOR and Specific Device Class i SIMPLE_GARAGE_DOOR.

There is no XML file generated in ./etc/zwave

I don't know if this is a "801" type of FGMS-001 (how do I find out?). According to Amazon, the Item Model Number is 857934005096.

Here is some debug logs:

```
01:45:08.615 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:44   ] - Setting controller into INCLUSION mode.
01:45:08.616 [DEBUG] [o.b.z.i.protocol.SerialMessage:113  ] - NODE 255: Creating empty message of class = AddNodeToNetwork (0x4A), type = Request (0x00)
01:45:08.617 [DEBUG] [b.z.i.protocol.ZWaveController:632  ] - Enqueueing message. Queue length = 1
01:45:08.617 [DEBUG] [WaveController$ZWaveSendThread:1241 ] - Took message from queue for sending. Queue length = 0
01:45:08.618 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 05 00 4A 81 FF CE 
01:45:08.619 [DEBUG] [WaveController$ZWaveSendThread:1301 ] - NODE 255: Sending REQUEST Message = 01 05 00 4A 81 FF CE 
01:45:08.626 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 07 00 4A FF 01 00 00 4C 
01:45:08.628 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:08.628 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:08.628 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 07 00 4A FF 01 00 00 4C 
01:45:08.630 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 07 00 4A FF 01 00 00 4C 
01:45:08.630 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 01 00 00 
01:45:08.631 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:77   ] - Add Node: Learn ready.
01:45:08.632 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveInclusionEvent
01:45:08.632 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:08.633 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = 81 FF 
01:45:08.633 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 01 00 00 
01:45:08.635 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=AddNodeToNetwork, expected=AddNodeToNetwork, cancelled=false
01:45:08.640 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:08.642 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:08.642 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:08.643 [DEBUG] [WaveController$ZWaveSendThread:1364 ] - NODE 255: Response processed after 23ms/113ms.
01:45:12.046 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 07 00 4A FF 02 00 00 4F 
01:45:12.052 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:12.052 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:12.053 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 07 00 4A FF 02 00 00 4F 
01:45:12.054 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 07 00 4A FF 02 00 00 4F 
01:45:12.054 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 02 00 00 
01:45:12.055 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:81   ] - Add Node: New node found.
01:45:12.060 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = 81 FF 
01:45:12.062 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 02 00 00 
01:45:12.063 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=AddNodeToNetwork, expected=AddNodeToNetwork, cancelled=false
01:45:12.063 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:12.064 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:12.064 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:12.219 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 1E 00 4A FF 03 06 17 04 07 01 5E 20 86 72 5A 59 85 73 84 80 71 56 70 31 8E 22 30 9C 98 7A BB 
01:45:12.221 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:12.221 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:12.223 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 1E 00 4A FF 03 06 17 04 07 01 5E 20 86 72 5A 59 85 73 84 80 71 56 70 31 8E 22 30 9C 98 7A BB 
01:45:12.224 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 1E 00 4A FF 03 06 17 04 07 01 5E 20 86 72 5A 59 85 73 84 80 71 56 70 31 8E 22 30 9C 98 7A BB 
01:45:12.225 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 03 06 17 04 07 01 5E 20 86 72 5A 59 85 73 84 80 71 56 70 31 8E 22 30 9C 98 7A 
01:45:12.226 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:84   ] - NODE 6: Adding slave.
01:45:12.226 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveInclusionEvent
01:45:12.227 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:12.227 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = 81 FF 
01:45:12.228 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 03 06 17 04 07 01 5E 20 86 72 5A 59 85 73 84 80 71 56 70 31 8E 22 30 9C 98 7A 
01:45:12.229 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=AddNodeToNetwork, expected=AddNodeToNetwork, cancelled=false
01:45:12.230 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:12.230 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:12.230 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:12.508 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 07 00 4A FF 05 06 00 4E 
01:45:12.511 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:12.511 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:12.511 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 07 00 4A FF 05 06 00 4E 
01:45:12.511 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 07 00 4A FF 05 06 00 4E 
01:45:12.512 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 05 06 00 
01:45:12.512 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:94   ] - Add Node: Protocol done.
01:45:12.513 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = 81 FF 
01:45:12.513 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 05 06 00 
01:45:12.514 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=AddNodeToNetwork, expected=AddNodeToNetwork, cancelled=false
01:45:12.515 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:12.515 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:12.516 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.623 [DEBUG] [.z.i.config.ZWaveConfiguration:1370 ] - Ending inclusion mode.
01:45:38.624 [DEBUG] [.z.i.config.ZWaveConfiguration:1393 ] - Stopping inclusion timer.
01:45:38.624 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:60   ] - Ending INCLUSION mode.
01:45:38.625 [DEBUG] [o.b.z.i.protocol.SerialMessage:113  ] - NODE 255: Creating empty message of class = AddNodeToNetwork (0x4A), type = Request (0x00)
01:45:38.625 [DEBUG] [b.z.i.protocol.ZWaveController:632  ] - Enqueueing message. Queue length = 1
01:45:38.625 [DEBUG] [WaveController$ZWaveSendThread:1241 ] - Took message from queue for sending. Queue length = 0
01:45:38.626 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 04 00 4A 05 B4 
01:45:38.627 [DEBUG] [WaveController$ZWaveSendThread:1301 ] - NODE 255: Sending REQUEST Message = 01 04 00 4A 05 B4 
01:45:38.783 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 07 00 4A FF 06 06 00 4D 
01:45:38.785 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:38.785 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:38.786 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 07 00 4A FF 06 06 00 4D 
01:45:38.786 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 07 00 4A FF 06 06 00 4D 
01:45:38.787 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 06 06 00 
01:45:38.787 [DEBUG] [.b.z.i.p.s.AddNodeMessageClass:97   ] - Add Node: Done.
01:45:38.788 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveInclusionEvent
01:45:38.788 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.789 [DEBUG] [.z.i.config.ZWaveConfiguration:1393 ] - Stopping inclusion timer.
01:45:38.789 [ERROR] [.z.i.config.ZWaveConfiguration:1399 ] - Neither inclusion nor exclusion was active!
01:45:38.790 [DEBUG] [b.z.i.protocol.ZWaveController:659  ] - NODE 6: Including node.
01:45:38.791 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = 05 
01:45:38.791 [DEBUG] [b.z.i.protocol.ZWaveController:506  ] - NODE 6: Init node thread start
01:45:38.791 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = AddNodeToNetwork (0x4A), type = Request (0x00), payload = FF 06 06 00 
01:45:38.792 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=AddNodeToNetwork, expected=AddNodeToNetwork, cancelled=false
01:45:38.792 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:38.793 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:38.793 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.793 [DEBUG] [WaveController$ZWaveSendThread:1364 ] - NODE 255: Response processed after 166ms/166ms.
01:45:38.797 [DEBUG] [.b.z.i.p.i.ZWaveNodeSerializer:138  ] - NODE 6: Serializing from file etc\zwave\node6.xml
01:45:38.804 [DEBUG] [.b.z.i.p.i.ZWaveNodeSerializer:141  ] - NODE 6: Error serializing from file: file does not exist.
01:45:38.805 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267  ] - NODE 6: Node advancer - EMPTYNODE: queue length(0), free to send(true)
01:45:38.807 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - EMPTYNODE try 1: stageAdvanced(false)
01:45:38.807 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:337  ] - NODE 6: Node advancer: Initialisation starting
01:45:38.808 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:890  ] - NODE 6: Node advancer - advancing to PROTOINFO
01:45:38.809 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - PROTOINFO try 0: stageAdvanced(true)
01:45:38.809 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:346  ] - NODE 6: Node advancer: PROTOINFO - send IdentifyNode
01:45:38.809 [DEBUG] [o.b.z.i.protocol.SerialMessage:113  ] - NODE 255: Creating empty message of class = IdentifyNode (0x41), type = Request (0x00)
01:45:38.810 [DEBUG] [b.z.i.protocol.ZWaveController:632  ] - Enqueueing message. Queue length = 1
01:45:38.810 [DEBUG] [WaveController$ZWaveSendThread:1241 ] - Took message from queue for sending. Queue length = 0
01:45:38.812 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240  ] - NODE 6: Node advancer - queued packet. Queue length is 1
01:45:38.812 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 04 00 41 06 BC 
01:45:38.813 [DEBUG] [b.z.i.protocol.ZWaveController:595  ] - NODE 6: Init node thread finished
01:45:38.813 [DEBUG] [WaveController$ZWaveSendThread:1301 ] - NODE 255: Sending REQUEST Message = 01 04 00 41 06 BC 
01:45:38.820 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 09 01 41 53 9C 01 04 07 01 7A 
01:45:38.827 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:38.827 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:38.829 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 09 01 41 53 9C 01 04 07 01 7A 
01:45:38.829 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 09 01 41 53 9C 01 04 07 01 7A 
01:45:38.830 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = IdentifyNode (0x41), type = Response (0x01), payload = 53 9C 01 04 07 01 
01:45:38.831 [DEBUG] [i.p.s.IdentifyNodeMessageClass:56   ] - NODE 6: ProtocolInfo
01:45:38.831 [DEBUG] [i.p.s.IdentifyNodeMessageClass:72   ] - NODE 6: Listening = false
01:45:38.832 [DEBUG] [i.p.s.IdentifyNodeMessageClass:73   ] - NODE 6: Routing = true
01:45:38.832 [DEBUG] [i.p.s.IdentifyNodeMessageClass:74   ] - NODE 6: Beaming = true
01:45:38.832 [DEBUG] [i.p.s.IdentifyNodeMessageClass:75   ] - NODE 6: Version = 4
01:45:38.833 [DEBUG] [i.p.s.IdentifyNodeMessageClass:76   ] - NODE 6: FLIRS = false
01:45:38.833 [DEBUG] [i.p.s.IdentifyNodeMessageClass:77   ] - NODE 6: Security = false
01:45:38.834 [DEBUG] [i.p.s.IdentifyNodeMessageClass:78   ] - NODE 6: Max Baud = 40000
01:45:38.834 [DEBUG] [i.p.s.IdentifyNodeMessageClass:94   ] - NODE 6: Basic = Routing Slave
01:45:38.834 [DEBUG] [i.p.s.IdentifyNodeMessageClass:102  ] - NODE 6: Generic = Garage Door
01:45:38.835 [DEBUG] [i.p.s.IdentifyNodeMessageClass:110  ] - NODE 6: Specific = Simple Garage Door
01:45:38.836 [DEBUG] [.o.b.z.i.p.c.ZWaveCommandClass:243  ] - NODE 6: Creating new instance of command class NO_OPERATION
01:45:38.836 [DEBUG] [.z.internal.protocol.ZWaveNode:613  ] - NODE 6: Adding command class NO_OPERATION to the list of supported command classes.
01:45:38.836 [DEBUG] [.o.b.z.i.p.c.ZWaveCommandClass:243  ] - NODE 6: Creating new instance of command class BASIC
01:45:38.837 [DEBUG] [.z.internal.protocol.ZWaveNode:613  ] - NODE 6: Adding command class BASIC to the list of supported command classes.
01:45:38.837 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = IdentifyNode (0x41), type = Request (0x00), payload = 06 
01:45:38.838 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = IdentifyNode (0x41), type = Response (0x01), payload = 53 9C 01 04 07 01 
01:45:38.838 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=IdentifyNode, expected=IdentifyNode, cancelled=false
01:45:38.839 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:38.839 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:38.839 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.840 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1034 ] - NODE 6: Node advancer - PROTOINFO: Transaction complete (IdentifyNode:Request) success(true)
01:45:38.840 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:202  ] - NODE 6: Node advancer - checking initialisation queue. Queue size 1.
01:45:38.841 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:208  ] - NODE 6: Node advancer - message removed from queue. Queue size 0.
01:45:38.841 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267  ] - NODE 6: Node advancer - PROTOINFO: queue length(0), free to send(true)
01:45:38.842 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - PROTOINFO try 1: stageAdvanced(false)
01:45:38.843 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:890  ] - NODE 6: Node advancer - advancing to NEIGHBORS
01:45:38.843 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - NEIGHBORS try 0: stageAdvanced(true)
01:45:38.844 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:356  ] - NODE 6: Node advancer: NEIGHBORS - send RoutingInfo
01:45:38.844 [DEBUG] [p.s.GetRoutingInfoMessageClass:30   ] - NODE 6: Request routing info
01:45:38.848 [DEBUG] [o.b.z.i.protocol.SerialMessage:113  ] - NODE 255: Creating empty message of class = GetRoutingInfo (0x80), type = Request (0x00)
01:45:38.849 [DEBUG] [b.z.i.protocol.ZWaveController:632  ] - Enqueueing message. Queue length = 1
01:45:38.849 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240  ] - NODE 6: Node advancer - queued packet. Queue length is 1
01:45:38.850 [DEBUG] [WaveController$ZWaveSendThread:1364 ] - NODE 255: Response processed after 37ms/166ms.
01:45:38.850 [DEBUG] [WaveController$ZWaveSendThread:1241 ] - Took message from queue for sending. Queue length = 0
01:45:38.851 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 07 00 80 06 00 00 03 7D 
01:45:38.852 [DEBUG] [WaveController$ZWaveSendThread:1301 ] - NODE 255: Sending REQUEST Message = 01 07 00 80 06 00 00 03 7D 
01:45:38.859 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 20 01 80 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 5F 
01:45:38.865 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:38.865 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:38.866 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 20 01 80 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 5F 
01:45:38.867 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 20 01 80 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 5F 
01:45:38.868 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = GetRoutingInfo (0x80), type = Response (0x01), payload = 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
01:45:38.868 [DEBUG] [p.s.GetRoutingInfoMessageClass:49   ] - NODE 6: Got NodeRoutingInfo request.
01:45:38.869 [DEBUG] [p.s.GetRoutingInfoMessageClass:79   ] - NODE 6: Neighbor nodes: 1
01:45:38.869 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveNetworkEvent
01:45:38.870 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.870 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = GetRoutingInfo (0x80), type = Request (0x00), payload = 06 00 00 03 
01:45:38.871 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = GetRoutingInfo (0x80), type = Response (0x01), payload = 01 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
01:45:38.871 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=GetRoutingInfo, expected=GetRoutingInfo, cancelled=false
01:45:38.872 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:38.872 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:38.873 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.873 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1034 ] - NODE 6: Node advancer - NEIGHBORS: Transaction complete (GetRoutingInfo:Request) success(true)
01:45:38.874 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:202  ] - NODE 6: Node advancer - checking initialisation queue. Queue size 1.
01:45:38.874 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:208  ] - NODE 6: Node advancer - message removed from queue. Queue size 0.
01:45:38.875 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267  ] - NODE 6: Node advancer - NEIGHBORS: queue length(0), free to send(true)
01:45:38.876 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - NEIGHBORS try 1: stageAdvanced(false)
01:45:38.876 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:890  ] - NODE 6: Node advancer - advancing to FAILED_CHECK
01:45:38.876 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - FAILED_CHECK try 0: stageAdvanced(true)
01:45:38.877 [DEBUG] [i.p.s.IsFailedNodeMessageClass:32   ] - NODE 6: Requesting IsFailedNode status from controller.
01:45:38.878 [DEBUG] [o.b.z.i.protocol.SerialMessage:113  ] - NODE 255: Creating empty message of class = IsFailedNodeID (0x62), type = Request (0x00)
01:45:38.878 [DEBUG] [b.z.i.protocol.ZWaveController:632  ] - Enqueueing message. Queue length = 1
01:45:38.878 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:240  ] - NODE 6: Node advancer - queued packet. Queue length is 1
01:45:38.878 [DEBUG] [WaveController$ZWaveSendThread:1364 ] - NODE 255: Response processed after 26ms/166ms.
01:45:38.879 [DEBUG] [WaveController$ZWaveSendThread:1241 ] - Took message from queue for sending. Queue length = 0
01:45:38.879 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 04 00 62 06 9F 
01:45:38.880 [DEBUG] [WaveController$ZWaveSendThread:1301 ] - NODE 255: Sending REQUEST Message = 01 04 00 62 06 9F 
01:45:38.886 [DEBUG] [eController$ZWaveReceiveThread:1530 ] - Receive Message = 01 04 01 62 00 98 
01:45:38.888 [DEBUG] [eController$ZWaveReceiveThread:1446 ] - Receive queue ADD: Length=1
01:45:38.888 [DEBUG] [b.z.i.protocol.ZWaveController:1194 ] - Receive queue TAKE: Length=0
01:45:38.888 [DEBUG] [o.b.z.i.protocol.SerialMessage:243  ] - Assembled message buffer = 01 04 01 62 00 98 
01:45:38.889 [DEBUG] [b.z.i.protocol.ZWaveController:1195 ] - Process Message = 01 04 01 62 00 98 
01:45:38.889 [DEBUG] [b.z.i.protocol.ZWaveController:194  ] - Message: class = IsFailedNodeID (0x62), type = Response (0x01), payload = 00 
01:45:38.890 [DEBUG] [i.p.s.IsFailedNodeMessageClass:56   ] - NODE 6: Is currently marked as healthy by the controller
01:45:38.892 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:66   ] - Sent message Message: class = IsFailedNodeID (0x62), type = Request (0x00), payload = 06 
01:45:38.892 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:67   ] - Recv message Message: class = IsFailedNodeID (0x62), type = Response (0x01), payload = 00 
01:45:38.892 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:68   ] - Checking transaction complete: class=IsFailedNodeID, expected=IsFailedNodeID, cancelled=false
01:45:38.892 [DEBUG] [.z.i.p.s.ZWaveCommandProcessor:74   ] -          transaction complete!
01:45:38.893 [DEBUG] [b.z.i.protocol.ZWaveController:648  ] - Notifying event listeners: ZWaveTransactionCompletedEvent
01:45:38.893 [DEBUG] [.z.internal.ZWaveActiveBinding:449  ] - ZwaveIncomingEvent
01:45:38.893 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:1034 ] - NODE 6: Node advancer - FAILED_CHECK: Transaction complete (IsFailedNodeID:Request) success(true)
01:45:38.894 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:202  ] - NODE 6: Node advancer - checking initialisation queue. Queue size 1.
01:45:38.894 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:208  ] - NODE 6: Node advancer - message removed from queue. Queue size 0.
01:45:38.895 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:267  ] - NODE 6: Node advancer - FAILED_CHECK: queue length(0), free to send(true)
01:45:38.900 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - FAILED_CHECK try 1: stageAdvanced(false)
01:45:38.900 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:890  ] - NODE 6: Node advancer - advancing to WAIT
01:45:38.901 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:332  ] - NODE 6: Node advancer: loop - WAIT try 0: stageAdvanced(true)
01:45:38.901 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:380  ] - NODE 6: Node advancer: WAIT - Listening=false, FrequentlyListening=false
01:45:38.902 [DEBUG] [z.i.p.i.ZWaveNodeStageAdvancer:398  ] - NODE 6: Node advancer: WAIT - Still waiting!
01:45:38.902 [DEBUG] [WaveController$ZWaveSendThread:1364 ] - NODE 255: Response processed after 22ms/166ms.

```

**Comment by cdjackson** [comment]
No data has been received from the device in this log, that's why it shows as unknown.

**Comment by pallemannen** [comment]
Hmm.. not sure I understand. Node 6 is the device. Some kind of data must have been received, or node 6 would never have been created.

Including and excluding works. I.e. if I put the controller in inclusion/exclusion mode, I can get node 6 to disappear (and reaappear as node 7) by pressing the button on the FGMS-001 device. It was once node 2. 

But Manufacturer ID and Device ID still says as UNKNOWN though.

**Comment by cdjackson** [comment]
> Some kind of data must have been received, or node 6 would never have been created.

Nope. No data FROM THE DEVICE has been received. The stick knows that the device exists, and the binding has communicated with the stick, and the stick has provided the basic information. This is as far as it goes, and the next communication to learn about the device and get the manufacturer etc comes from the device and this has not happened.

**Comment by pallemannen** [comment]
Hi Chris

Ah. So (if I understand correctly) the device has sent data to the stick (at least "I exist"), but nothing to OpenHAB. And it hasn't sent the Manufacturer ID and Device ID at all (at least not in a way that would reach OpenHAB).

Anyway - I went to sleep yesterday. As you can see in the log, that was around 0145. Two hours later, at 0345, the device decided it was time to let us know who he was. Now Habmin recognizes it as the correct model.

And - it was the 801 type. :)

**Comment by cdjackson** [comment]
Yes - during the inclusion this information gets sent to the device.

So, without reading back through this long thread, is 801 good, and your device is working fine now?

**Comment by pallemannen** [comment]
Chris: Not sure yet. It looks good so far, but I haven't tested any sensors yet.

The association groups and configuration parameters got populated when the device finally woke up, so I could configure all sensors to report to node 1 and change some other parameters.

The wake up period was 7200 - which I guess correlates with 0145 to 0345. It seems a bit impractical that an inclusion should take two hours to complete though.

**Comment by cdjackson** [comment]
On OH1, if the config/associations get set, then there’s not much else to do, so should be fine.

**Comment by pallemannen** [comment]
Yep:

![image](https://cloud.githubusercontent.com/assets/8362176/17638524/56580c08-609f-11e6-919b-e9c20ccd8a94.png)

**Comment by xdxoxkx** [comment]
@pallemannen 
Can you please post your "items" code - i have still problems with Motion. Thx!

**Comment by pallemannen** [comment]
Number eye_lux "Lux: [%.2f Lux]" <zwave> (hepp) {zwave="6:command=sensor_multilevel,sensor_type=3"}
Number eye_temp "Temp: [%.1f °C]" <zwave> (hepp) {zwave="6:command=sensor_multilevel,sensor_type=1"}
Contact eye_motion "Motion: [%s]" <zwave> (hepp) {zwave="6:command=sensor_binary"}
Number eye_batt "Battery: [%d %%]" <zwave> (hepp) {zwave="6:command=battery"}
Switch eye_basic "Basic: [%s]" <zwave> (hepp) {zwave="6:command=basic"}
Contact eye_alarm "Alarm: [%s]" <zwave> (hepp) {zwave="6:command=alarm"}

**Comment by Gilles76** [comment]
Hi Chris,
I installed a fgms001 to my family in law, device id 801 and never get sensor_binary class whereas it was set up in configurations.

Everything is ok except the generic class and specific class.

Generic class is garage_door and specific class is simple_garage_door.

Waiting a long time (more than 1 week) don't do anything.

`<deviceClass>
    <basicDeviceClass>ROUTING_SLAVE</basicDeviceClass>
    <genericDeviceClass>BINARY_SENSOR</genericDeviceClass>
    <specificDeviceClass>ROUTING_SENSOR_BINARY</specificDeviceClass>
  </deviceClass>
  <homeId>XXXXXXXX</homeId>
  <nodeId>9</nodeId>
  <version>4</version>
  <manufacturer>0x10f</manufacturer>
  <deviceId>0x1001</deviceId>
  <deviceType>0x801</deviceType>
  <listening>false</listening>
  <frequentlyListening>false</frequentlyListening>
  <routing>true</routing>
  <security>false</security>
  <beaming>true</beaming>
  <maxBaudRate>40000</maxBaudRate>
  <nodeInformationFrame/>
  <supportedCommandClasses>
`

What do you need to solve this issue ?

Gilles.

**Comment by pallemannen** [comment]
xdxoxkx: I also added the below. Don't know if it works though - still waiting for my first quake.

Number eye_seismic "Seismic: [%d]" {zwave="6:command=sensor_multilevel,sensor_type=25"}

Gilles76: It will eventually be changed in the code, just ignore till then.

**Comment by pallemannen** [comment]
I became the earth quake myself, and it worked fine:

2016-08-15 23:32:23 - eye_seismic state updated to 5.8
2016-08-15 23:32:53 - eye_seismic state updated to 0

**Comment by pallemannen** [comment]
Correction: Number eye_seismic "Seismic: [**%.1f**]" {zwave="6:command=sensor_multilevel,sensor_type=25"}

**Comment by xdxoxkx** [comment]
I still have trouble. On Motion I get "ALARM" and not "MOTION":

2016-08-16 08:32:13.406 [INFO ] [runtime.busevents             ] - OG_Akleide_Sensor_Alarm state updated to CLOSED

Latest z-wave jar.

Do I have to modify the Association Groups? 
Any hint?

**Comment by anordvall** [comment]
I belive you don't really have the latest z-wave jar. I had the same problem before before updating.

Go to https://openhab.ci.cloudbees.com/job/openHAB1-Addons/lastSuccessfulBuild/artifact/bundles/binding/org.openhab.binding.zwave/target/

... and download the file to your addons folder. Even though you already have version 1.9.0 you might be using an old build of it.

**Comment by pallemannen** [comment]
On motion, I get MOTION and BASIC. I get ALARM=OFF on tamper - AND periodically for no reason. I never get ALARM=ON.

So removed the BASIC and ALARM items, detect motion on MOTION and detect earth quakes on SENSOR_MULTILEVEL 25 reports above 0. The temp, lux and battery reports behaves as expected. I don't care about tamper, even though it's located in my kid's room.

**Comment by Gilles76** [comment]
@ Pallemannen,

Here is what  I get for motion sensor: 

`NODE 9: Got a value event from Z-Wave network, endpoint = 0, command class = BASIC, value = 255
2016-08-16 14:23:16.476 WARN  o.o.b.z.i.ZWaveActiveBinding[:485]- NODE 9: No item bound for event, endpoint = 0, command class = BASIC, value = 255, ignoring.
`
I'm interested in motion detection.

my item is:
`zwave="9:0:command=basic,respond_to_basic=true"`

It's not a good setup as the error message. I will try with endpoint alone.

**Comment by xdxoxkx** [comment]
@anordvall 
Thx for your help - my zwave jar was from yesterday, I took yours now ... nothing changed

In the Association Groups (there are 5) under "Motion BC" I set "Node 1" as a member. 
Now I get on 1 movement:
OG_Ankleide_Sensor_Alarm state updated to CLOSED
2016-08-16 15:02:00.027 [INFO ] [runtime.busevents             ] - OG_Ankleide_Sensor_Motion state updated to OPEN
2016-08-16 15:02:00.046 [INFO ] [runtime.busevents             ] - OG_Ankleide_Sensor_Motion state updated to OPEN

I have 50 Z-Wave components ... no problems with items ... but this one is really strange for me.

**Comment by Gilles76** [comment]
Well, it's working.
For having motion sensorworking , my setup is:
for item:
`zwave="9:0"`

for sitemap:
`Switch item=Motion`

As the sensor send basic information for motion, the item must be a switch as a number item isn't working.

So effectively, the bad class seen in habmin's information isn't important.

Hope that helps.

Gilles.

**Comment by pallemannen** [comment]
This is what I have and works:

Number eye_lux "Lux: [%.2f Lux]" {zwave="6:command=sensor_multilevel,sensor_type=3"}
Number eye_temp "Temp: [%.1f °C]" {zwave="6:command=sensor_multilevel,sensor_type=1"}
Number eye_seismic "Seismic: [%.1f]" {zwave="6:command=sensor_multilevel,sensor_type=25"}
Contact eye_motion "Motion: [%s]" {zwave="6:command=sensor_binary"}
Number eye_batt "Battery: [%d %%]" {zwave="6:command=battery"}

I also get BASIC reports - and they are always duplicates of the sensor_binary one, so I removed it from the configuration.

Anybody know if the Seismic sensor is reporting the richter scale or something else?

**Comment by TimHaydnJones** [comment]
pallemannen - from http://manuals.fibaro.com/content/manuals/en/FGMS-001/FGMS-001-EN-T-v2.0.pdf

Seismograph mode:
The Motion Sensor can be configured to work as a simple earthquake
detector, by setting the parameter 24 value to 4. Reports with scale of
the vibrations (in the **Modified Mercalli Scale**) will be sent at the time
intervals specified in parameter 20. First report will be sent immediately
after vibrations have been detected. Once the vibrations cease,
reports will stop being sent.
